webdriver-spec.html

```diff
@@ -1968,42 +1968,72 @@
  </tr>
 </table>
 
-<p>The <dfn>Status</dfn> <a>command</a> returns information about
-  whether a remote end is in a state in which it can create a new
-  session. This is represented by the <code>ready</code> property of the
-  response body, which has a value of <code>false</code> if attempting
-  to create a session at the current time would fail. A value
-  of <code>true</code> does not guarantee a new session request would
-  succeed.
+<p>The <dfn>Status</dfn> <a>command</a>
+ returns information about whether a <a>remote end</a>
+ is in a state in which it can create <a data-lt="new session">new sessions</a>
+ and can additionally include arbitrary meta information
+ that is specific to the implementation.
+
+<p>The readiness state is represented
+ by the <code>ready</code> property of the body,
+ which is false if an attempt to <a data-lt="new session">create a session</a>
+ at the current time would fail.
+ However, the value true does not guarantee
+ that a new session request will succeed.
+
+<p>Implementations may optionally include
+ arbitrary other meta information as part of the body,
```

Comment:
Nit: No need for "arbitrary" here, as you've already said that this is the case above.